### PR TITLE
Footer link fixes

### LINKED ISSUE #18: Visual bugs after merged PR #4
After merged PR #4 the following visual bugs can be observed:

- the is no space between links in the footer on some screen breakouts
- some links have underline on hover, some not
- internal links should not open in the new window

screenshot 2019-02-06 at 09 40 59

### PR DESCRIPTION
**Pull request by Michadelic**
* harmonized footer links
* added margin between footer links
* google analytics disclaimer does not open in new tab anymore

Fixes #18

### CODE REVIEW

File: src/pages/community.html [hunk 1]
```diff
@@ -167,10 +167,10 @@
 			<div class="footer">
 				<div class="footer__content">
 					<div class="footer__item">&copy; 2019 SAP SE. Made available under <nobr>Apache License 2.0</nobr></div>
-					<div class="footer__item"><a href="https://www.sap.com/about/legal/impressum.html" rel="noopener" target="_blank">Legal Disclosure</a></div>
+					<div class="footer__item"><a href="https://www.sap.com/about/legal/impressum.html" rel="noopener" target="_blank" class="link">Legal Disclosure</a></div>
 					<div class="footer__item"><a href="https://www.sap.com/corporate/en/legal/terms-of-use.html" rel="noopener" target="_blank" class="link">Terms of Use</a></div>
-					<div class="footer__item"><a href="https://www.sap.com/about/legal/privacy.html" rel="noopener" target="_blank">Privacy</a></div>
-					<div class="footer__item"><a href="../ga_disclaimer/" rel="noopener" target="_blank" class="link">Google Analytics Disclaimer</a></div>
+					<div class="footer__item"><a href="https://www.sap.com/about/legal/privacy.html" rel="noopener" target="_blank" class="link">Privacy</a></div>
+					<div class="footer__item"><a href="../ga_disclaimer/" rel="noopener" class="link">Google Analytics Disclaimer</a></div>
```
Comment (pavelkornev) [inline src/pages/community.html]: please remove `rel="noopener"` for the same origin links.

File: src/pages/community.html [hunk 2]
```diff
@@ -167,10 +167,10 @@
 			<div class="footer">
 				<div class="footer__content">
 					<div class="footer__item">&copy; 2019 SAP SE. Made available under <nobr>Apache License 2.0</nobr></div>
-					<div class="footer__item"><a href="https://www.sap.com/about/legal/impressum.html" rel="noopener" target="_blank">Legal Disclosure</a></div>
+					<div class="footer__item"><a href="https://www.sap.com/about/legal/impressum.html" rel="noopener" target="_blank" class="link">Legal Disclosure</a></div>
```
Comment (pavelkornev) [inline src/pages/community.html]: please remove unused CSS:
https://github.com/SAP/openui5-website/blob/master/src/components/footer/styles.css#L21:L29

Comment (Michadelic) [reply src/pages/community.html]: which part is unused?

Comment (pavelkornev) [reply src/pages/community.html]: is it marked by the link above? Lines 21-29:
screenshot 2019-02-14 at 12 33 57

**Comment by pavelkornev** [comment]
This PR was closed by mistake. Apologise for this.

**Comment by Michadelic** [comment]
to my knowledge, the referenced CSS classes are still used in the footer area:
footercss

**Comment by pavelkornev** [comment]
@Michadelic I've cleaned CSS myself, but the originally reported problem is still there:
screenshot 2019-02-14 at 15 51 54

**Comment by Michadelic** [comment]
fixed

**Comment by pavelkornev** [comment]
@Michadelic, the alignment of the copyright is not centered properly. There is no such problem in the current version on openui5.org, so it's a regression:

margins

**Pull request merged**